packages/backend/src/server/web/boot.js

```diff
@@ -104,20 +104,27 @@
 
 	// eslint-disable-next-line no-inner-declarations
 	function renderError(code, details) {
-		document.documentElement.innerHTML = `
-			<h1>⚠エラーが発生しました</h1>
-			<p>問題が解決しない場合は管理者までお問い合わせください。以下のオプションを試すこともできます:</p>
+		let errorsElement = document.getElementById('errors');
```

Comment:
I think it could be a problem if another element in Misskey would accidentally get `id="errors"`. We would always have to keep it in mind, tricky...

Reply:
Maybe give it the id of something like `bios-errors` or `bios-errors-on-boot` so it can't be messed up

Reply:
これが表示された後にMisskey (Vue App)が展開されることはまずない、というか、init.tsはVue Appが展開される前にエラーハンドラを削除するのでそういうことは多分ない

（そこまで気にするならもうちょっといい感じに書かなきゃいけないけど、それは面倒）

Reply:
ごちゃごちゃ書いたからちゃんと機械翻訳されない気がするけど、とにかく、そこまで気にしてここを書くのが面倒と私は思っている

Reply:
> It is unlikely that Misskey (Vue App) will be deployed after this is displayed, or perhaps not, since init.ts removes the error handler before Vue App is deployed.

Ah, I did not realize the error handler gets removed. In that case it should not be a problem.

> I have a feeling it won't be machine translated properly.

Seems fine.